check-module-version/action.yml

```diff
@@ -0,0 +1,51 @@
+name: 'Check module version'
+description: >
+  Сhecks the equality of the version from Lua 
+  module's code and the last repository tag
+
+inputs:
+  module-name:
+    description: 'Lua module name as when using require() in tarantool'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Clone the module
+      uses: actions/checkout@v3
```

Comment:
Can modules have submodules?

Reply:
Yep, in general, there may be such modules, so now I added `recursive` to checkout step